Nix-Shell.md

```diff
@@ -0,0 +1,17 @@
+# Nix Shell
+Nix development shells can be used to get development environments which can be accessed using the `nix-shell` command.
```

Comment:
I think this is still lacking context to understand it. Is there Nix documentation you could link to that goes into more depth about what it means to have a nix development shell or what the benefits are?

Reply:
I'll get on that soon, have had other projects to work on.